plugins/hydra_ax_sweeper/tests/config/quadratic.yaml

```diff
@@ -0,0 +1,36 @@
+defaults:
+  - hydra/sweeper: ax
+
+quadratic:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
+
+    max_trials: 10
+    
+    experiment:
+      # Quadratic function should be minimized (Ax defaults to maximize)
+      minimize: true
+
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 2
+        
+    
+    params:
+      quadratic.x:
+        type: range
+        bounds: [-1, 1]
+        log_scale: false
+      
+      quadratic.y:
+        type: range
+        bounds: [-1, 1]
+        log_scale: false
+
```

Comment:
this is showcasing why it's good to make it optional.
There is a lot of unneeded boilerplate here.